src/client/driver/driver-link/window/child-window-closed-before-switching-error.js

```diff
@@ -0,0 +1,8 @@
+export default class ChildWindowClosedBeforeSwitchingError extends Error {
+    constructor () {
+        super();
+
+        // NOTE: https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work
```

Comment:
Due to this limitation on the client side, we use the special [isTestCafeError](https://github.com/DevExpress/testcafe/blob/master/src/errors/test-run/index.js#L11) property to detect our custom errors. So maybe it's better to subclass `TestRunErrorBase` to avoid messing with prototypes directly?

Reply:
`ChildWindowClosedBeforeSwitchingError` inherits the `Error` class only for semantic purposes.
`TestRunErrorBase` contains the `code`, `callsite` and `isTestCafeError` properties. I will not use one of these properties.
So, I propose to not inherit the `ChildWindowClosedBeforeSwitchingError` class from any class at all.
```js
export default class ChildWindowClosedBeforeSwitchingError {
}
```
That do you think about it?

Reply:
ping @AndreyBelym